### PR TITLE
Getting the repo building again

### PR DESCRIPTION
**Pull request by Petermarcu**
Fixing a few things up to get things building again.

### CODE REVIEW

**Comment by mellinoe** [comment]
Is this all we need? I needed to make a lot more changes to get the tests building, for example. `dotnet test` doesn't seem to work without quite a few more changes, i.e.:
https://github.com/dotnet/corefxlab/compare/master...mellinoe:dotnet-update

File: NuGet.Config [hunk 1]
```diff
@@ -16,6 +16,7 @@
     <add key="dotnet.myget.org dotnet-corefxlab" value="https://dotnet.myget.org/F/dotnet-corefxlab/" />    
     <add key="nuget.org" value="https://api.nuget.org/v3/index.json" protocolVersion="3" />
     <add key="dotnet.myget.org dotnet-cli" value="https://dotnet.myget.org/F/dotnet-cli/" />
+    <add key="dotnet.myget.org cli-deps" value="https://dotnet.myget.org/F/cli-deps/api/v3/index.json" />
```
Comment (livarcocc) [inline NuGet.Config]: We shouldn't have other repos depending on this feed. Whatever packages this repo depends on, we should be explicit and take a dependency on their original feeds.

cc @eerhardt

Comment (eerhardt) [inline NuGet.Config]: Agreed.  Only the "cli" repo should depend on "cli-deps".

@Petermarcu - which package(s) did you need this for?

**Comment by MichalStrehovsky** [comment]
Can we just pin the last known good version of the CLI and be done with it for a few months? We're doing that in the CoreRT repo because the CLI makes breaking changes all the time and it's a waste of engineering resources trying to keep up.

**Comment by mellinoe** [comment]
This should be superseded by the work I did.

**Pull request closed without merging**